[1.315s] Invoking command in '/home/yamashita/natu2024/build/points_msg': CMAKE_PREFIX_PATH=/home/yamashita/natu2024/install/autoRobo:/opt/ros/humble /usr/bin/cmake /home/yamashita/natu2024/src/points_msg -DCMAKE_INSTALL_PREFIX=/home/yamashita/natu2024/install/points_msg
[1.362s] -- Found ament_cmake: 1.3.10 (/opt/ros/humble/share/ament_cmake/cmake)
[1.973s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[1.980s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[1.993s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[2.164s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[3.055s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[3.056s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[3.056s]   points_msg --arguments-file
[3.057s]   /home/yamashita/natu2024/build/points_msg/rosidl_adapter__arguments__points_msg.json
[3.058s]   --output-dir
[3.058s]   /home/yamashita/natu2024/build/points_msg/rosidl_adapter/points_msg
[3.059s]   --output-file
[3.059s]   /home/yamashita/natu2024/build/points_msg/rosidl_adapter/points_msg.idls)
[3.060s]   returned error code 1:
[3.060s] 
[3.061s]   Error processing 'float64*[2] data' of 'points_msg/map_points': ''float64*'
[3.061s]   is an invalid message name.  It should have the pattern
[3.062s]   '^[A-Z][A-Za-z0-9]*$''
[3.063s] 
[3.063s]   Traceback (most recent call last):
[3.064s] 
[3.066s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[3.067s]       return _run_code(code, main_globals, None,
[3.068s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[3.069s]       exec(code, run_globals)
[3.070s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[3.071s]       sys.exit(main())
[3.072s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[3.074s]       abs_idl_file = convert_to_idl(
[3.075s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[3.076s]       return convert_msg_to_idl(
[3.077s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[3.078s]       msg = parse_message_string(package_name, input_file.stem, content)
[3.081s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[3.084s]       Type(type_string, context_package_name=pkg_name),
[3.084s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[3.085s]       super(Type, self).__init__(
[3.085s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[3.085s]       raise InvalidResourceName(
[3.086s] 
[3.086s]   rosidl_adapter.parser.InvalidResourceName: 'float64*' is an invalid message
[3.087s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[3.088s] 
[3.088s] Call Stack (most recent call first):
[3.088s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[3.089s]   CMakeLists.txt:13 (rosidl_generate_interfaces)
[3.089s] 
[3.090s] [0m
[3.090s] -- Configuring incomplete, errors occurred!
[3.091s] See also "/home/yamashita/natu2024/build/points_msg/CMakeFiles/CMakeOutput.log".
[3.092s] Invoked command in '/home/yamashita/natu2024/build/points_msg' returned '1': CMAKE_PREFIX_PATH=/home/yamashita/natu2024/install/autoRobo:/opt/ros/humble /usr/bin/cmake /home/yamashita/natu2024/src/points_msg -DCMAKE_INSTALL_PREFIX=/home/yamashita/natu2024/install/points_msg
